tests/framework/utils_cpuid.py

```diff
@@ -11,7 +11,7 @@
 from framework.utils_imdsv2 import imdsv2_get
 import host_tools.network as net_tools
 
-ARM_CPU_DICT = {"0xd0c": "ARM_NEOVERSE_N1"}
+ARM_CPU_DICT = {"0xd0c": "ARM_NEOVERSE_N1", "0xd40": "Graviton3"}
```

Comment:
To keep the naming scheme, I think this should be ARM_NEOVERSE_V1 https://en.wikipedia.org/wiki/AWS_Graviton#Graviton3

Reply:
Ok, didn't know that

Reply:
fixed